omnigibson/transition_rules.py

```diff
@@ -323,25 +323,48 @@ def __init__(self, filter_1_name, filter_2_name):
         # Will be filled in during self.initialize
         # Maps object to the list of rigid body idxs in the global contact matrix corresponding to filter 1
         self._filter_1_idxs = None
-        # List of rigid body idxs in the global contact matrix corresponding to filter 2
+        # Maps object to the list of rigid body idxs in the global contact matrix corresponding to filter 2
         self._filter_2_idxs = None
+        # Maps object to set of rigid bodies corresponding to filter 2
+        self._filter_2_bodies = None
```

Comment:
nit: maybe add a comment here saying `self. _filter_2_idxs` will be used for the optimized mode and `self. _filter_2_bodies` will be used for the non-optimized mode.

Reply:
done!